app/root.tsx

```diff
@@ -238,9 +236,10 @@ function UserDropdown() {
 							action="/logout"
 							method="POST"
 							className="rounded-b-3xl px-7 py-5 outline-none radix-highlighted:bg-night-500"
-							onClick={e => submit(e.currentTarget)}
```

Comment:
Unfortunately this change makes the form not keyboard accessible. Also, if we do this we'll want to move the padding from the form to the button so the entire row is clickable.